src/common/Header.js

```diff
@@ -14,14 +17,25 @@ const StyledToolbar = styled(Toolbar)(({ theme }) => ({
   backgroundColor: '#FAFAFA',
 }));
 
-export default function Header(props) {
+export default function Header() {
+  const location = useLocation();
+  const isAdmin = location.pathname.includes('admin');
+  const isPatient = location.pathname.includes('patient');
+  const { data: patientData, isFetching: patientFetching } = useGetPatientInfoQuery();
```

Comment:
I'm concerned with how many times we make this call (gas prices + transaction times). In restricting routes, it might be better if we make the one call here and then pass it down as props to admin & patient dashboards. Just a future issue, and more of an enhancement for optimization.

Reply:
Not sure if reading takes gas though - but good point for optimization